tests/MultiplexingDistributedCache.Tests/MultiplexingDistributedCacheTests.cs

```diff
@@ -0,0 +1,52 @@
+using Xunit;
+using Aqovia.Cache;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
+using Moq;
+using System;
+
+namespace Aqovia.Cache.Tests;
+
+public class MultiplexingDistributedCacheTests
+{
+    [Fact]
+    public void Primary_Get_Is_Called_Once()
+    {
+        var primary = new Mock<IDistributedCache>();
+        var secondary = new Mock<IDistributedCache>();
+
+        var cache = new MultiplexingDistributedCache(primary.Object, secondary.Object);
+
+        cache.Get("foo");
+
+        primary.Verify(_ => _.Get("foo"), Times.Once());
+    }
+
+    [Fact]
+    public void Secondary_Set_Is_Called_Once()

```

Comment:
I think we should add tests for the other operations too. Maybe it's possible to write a test that iterates through he option flavours - but it might be easier to just/copy paste. I think for async we also need to look at making sure that they complete.

Reply:
If you mean by Reflection, it's not easy to Verify the result. I will copy paste.

Reply:
I added more tests, but I couldn't add tests for completion of Async methods.